src/Templates/src/templates/maui-mobile/Resources/Styles/Styles.xaml

```diff
@@ -325,32 +325,30 @@
     </Style>
 
     <Style TargetType="Switch">
-        <Setter Property="OnColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}" />
-        <Setter Property="ThumbColor" Value="{StaticResource White}" />
```

Comment:
Do we no longer need the "default" or "normal" colors? Does Stephane's PR fix this old bug?

Reply:
not sure which of Stephane's PR you're referring to?
I got rid of it because "normal" visual states is a bit confusing for Switch since "on" and "off" have different styles